hack/local-check-gh-automation.sh

```diff
@@ -29,8 +29,14 @@ OC extract configmap/config --to "${data}/prow"
 
 app_id=$(cat "${data}/appid")
 
+candidate_path=''
+if [ ! -z "${2+x}" ]; then
+  candidate_path="--candidate-path=$2"
+fi
```

Comment:
Can we accept this as a named parameter, rather than relying on positional args?